TESTING.md

```diff
@@ -99,295 +99,301 @@ Marked with a github issue label "manual testing."
+**8.3** ASSERTION: Browser opens your profile page in A+ LMS (login, if asked).
 
-**9.4** Copy **API Access Token** from the web page and insert it into the prompt, click "OK".
+**8.4** Copy **API Access Token** from the web page and insert it into the prompt, click **OK**.
 
-**9.5** ASSERTION: The assignments tab is populated with Weekly assignments.
+**8.5** ASSERTION: The assignments tab is populated with weekly assignments.
 
 ![Assignments shown](images/assignments-shown.png)
 
-**9.6** Close the IDE.
+**8.6** Close the IDE.
 
-**9.7** Open the IDE with the project.
+**8.7** Open the IDE with the project.
  
-**9.8** If [safe password storage](https://jetbrains.org/intellij/sdk/docs/basics/persisting_sensitive_data.html?search=sensitive#storage) 
+**8.8** ASSERTION: Assignments tab is still populated. If [safe password storage](https://jetbrains.org/intellij/sdk/docs/basics/persisting_sensitive_data.html?search=sensitive#storage) 
 is activated in the OS.
 
-**9.9** ASSERTION: Assignments tab is still populated.
-
-### 10 Assignments
+### 9 Assignments submission
 
-#### Part I: Submitting an Assignment.
-
-**10.1** Expand **Week 1** and select **Assignment 2 (GoodStuff)**, and hit 
+**9.1** Expand **Week 1** and select **Assignment 2 (GoodStuff)**, and hit 
 **Submit A+ Assignment**.
 
 ![Submit](images/submit_button.png)
 
-**10.2** ASSERTION: The "Could not find module: The A+ Courses plugin could not find the 
-module GoodStuff." warning is shown.
+**9.2** ASSERTION: The "Could not find module: The A+ Courses plugin could not find the 
+module GoodStuff." error is shown.
 
-**10.3** Go to **Modules** tab and import **GoodStuff** module like described in **6.1**
+**9.3** Go to **Modules** tab and import **GoodStuff** module like described in **6.1**
 
-**10.4** Repeat **10.1**.
+**9.4** Repeat **9.1**.
 
-**10.5** ASSERTION: **Submit Assignment** dialog is shown.
+**9.5** ASSERTION: **Submit Assignment** dialog is shown.
 
-**10.6** Select **Submit as: > Submit alone**, click **OK**.
+**9.6** Select **Submit as: > Submit alone**, click **OK**.
 
-**10.7** ASSERTION: The "Assignment successfully sent for assessment: You will be notified here when
+**9.7** ASSERTION: The **Assignment successfully sent for assessment: You will be notified here when
  feedback is available. (You may also always check any of your submissions on the course website in 
- A+.)" notification is shown.
- 
-#### Part II. Checking the submission.
+ A+.)** notification is shown.
  
-**10.8** Wait a while.
+### 10 Refreshing Modules and Assignments
 
-**10.9** ASSERTION: The "Submission feedback available Feedback for Assignment 2 (Goodstuff) is now 
-available. Click the link below to open the feedback." notification is shown.
+**10.1** Hit the **Refresh Assignments** or **Refresh Modules** button.
+
+![Refresh](images/refresh.png)
+
+**10.2** ASSERTION: Assignments list is collapsed.
+ 
```

Comment:
I think this manual testing doc should reflect the desired/expected (by a user) behavior of the software rather than its actual implementation. This part makes little sense to a user as the list collapsing is more of an inconvenience than what is desired to happen when refreshing. (Of course for us, it's a sign that refresh was actually triggered but only because we know how that's been implemented.)

Instructions like this (that are based on implementation details) can also become easily outdated by some future UI improvements without developers realizing they are breaking manual testing workflow.

Reply:
So I would change the last assertion to be that the new submission has appeared there and add a note (in parentheses, maybe) that the assignment list may be collapsed when refresh button is pressed, but that's ok.